services-camel/src/test/java/edu/unc/lib/dl/services/camel/enhancements/EnhancementRouterIT.java

```diff
@@ -268,13 +304,20 @@ public void testProcessFilterOutDescriptiveMDSolr() throws Exception {
         verify(solrIngestProcessor, never()).process(any(Exchange.class));
     }
 
-    private static Map<String, Object> createEvent(PID pid, String... type) {
-
+    private Map<String, Object> createEvent(boolean editThumb, PID pid, String... type) {
         final Map<String, Object> headers = new HashMap<>();
         headers.put(IDENTIFIER, pid.getRepositoryPath());
         headers.put(EVENT_TYPE, "ResourceCreation");
         headers.put("CamelFcrepoUri", pid.getRepositoryPath());
         headers.put(RESOURCE_TYPE, String.join(",", type));
+        headers.put(CdrBinaryMimeType, "text/plain");
```

Comment:
I think this and the `if (editThumb)` should be removed, since the router and its processes should be setting these headers. You should also be able to remove the `boolean editThumb` parameter from this method